FirebaseFirestore.podspec

```diff
@@ -49,6 +49,10 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
   ]
   s.public_header_files = 'Firestore/Source/Public/*.h'
 
+  # TODO(varconst): remove once https://github.com/grpc/grpc/pull/16962 makes it
+  # into a release.
+  s.resource_bundles = { 'gRPCCertificates' => ['Firestore/core/src/firebase/firestore/remote/roots.pem'] }
```

Comment:
Minor nit on the file location: I'm not sure that putting the certs in the src directories is the best spot for it... I wouldn't think to look there myself.

Looking at grpc, they put this file in ./etc/roots.pem, which might imply Firestore/etc/roots.pem for us?

Optional.

Reply:
FWIW I wouldn't look to strongly at the location of `CAcert.pem` -- this was done as a test-only convenience and isn't intended to be a part of the shipping product that an end-user might want to change. This case is different so it seems reasonable to make this more obvious.

Reply:
Ok, moved to `Firestore/etc/roots.pem` -- let me know if another location is preferable.